### ISSUE TITLE
Stop using `settings.SENTRY_URL_PREFIX`

### ISSUE THREAD

**Issue by mattrobenolt** [getsentry/sentry-hipchat-ac#18]
Use `options.get('system.url-prefix')` instead. Make sure to handle the fact that this may be an empty string when not set up yet, so fail gracefully in that case. This value can change at runtime as well, so don't pre-compute and store a value somewhere since it can/will change.

**Comment by dcramer** [comment]
Is this a blocker? If so theres any number of other integrations that might have this issue too.

**Comment by mattrobenolt** [comment]
Nope, it's just potentially undesired results since the value won't get updated at runtime.

**Comment by dcramer** [comment]
Deprioritizing this as SENTRY_URL_PREFIX needs to be maintained at least for the 8.0 release.  GH-19 is the real issueh ere.
